go/consensus/tendermint/apps/supplementarysanity/checks.go

```diff
@@ -234,6 +245,51 @@ func checkKeyManager(ctx *abciAPI.Context, now epochtime.EpochTime) error {
 	return nil
 }
 
+func checkGovernance(ctx *abciAPI.Context, epoch epochtime.EpochTime) error {
+	st := governanceState.NewMutableState(ctx.State())
+
+	params, err := st.ConsensusParameters(ctx)
+	if err != nil {
+		return fmt.Errorf("ConsensusParameters: %w", err)
+	}
+	err = params.SanityCheck()
+	if err != nil {
+		return fmt.Errorf("SanityCheck ConsensusParameters: %w", err)
+	}
+	// Sanity check proposals.
```

Comment:
There should be a sanity check somewhere that the amount in the governance deposits address matches the total deposits of all active proposals.

Reply:
good idea! added it to the sanity checks (it's checked both in genesis and in the supplementary checker)